src/Workspaces/VisualBasicTest/VisualBasicSyntaxFactsServiceTests.vb

```diff
@@ -0,0 +1,221 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis.Text
+Imports Roslyn.Test.Utilities
+Imports Xunit
+
+Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
+    Public Class VisualBasicSyntaxFactsServiceTests
+
+        <Fact, WorkItem(4311, "https://github.com/dotnet/roslyn/issues/4311")>
+        Public Sub IsMethodLevelMember_Field()
+            Assert.True(IsMethodLevelMember("
+Class C
+    [|Dim x As Integer|]
+End Class"))
+        End Sub
+
+        <Fact, WorkItem(4311, "https://github.com/dotnet/roslyn/issues/4311")>
+        Public Sub IsMethodLevelMember_AutoProperty()
+            Assert.True(IsMethodLevelMember("
+Class C
+    [|Property x As Integer|]
+End Class"))
+        End Sub
+
+        <Fact, WorkItem(4311, "https://github.com/dotnet/roslyn/issues/4311")>
+        Public Sub IsMethodLevelMember_NormalProperty()
+            Assert.True(IsMethodLevelMember("
+Class C
+    [|Property x As Integer
+        Get
+            Return 42
+        End Get
+        Set (value As Integer)
+        End Set
+    End Property|]
+End Class"))
+        End Sub
+
+        <Fact, WorkItem(4311, "https://github.com/dotnet/roslyn/issues/4311")>
+        Public Sub IsMethodLevelMember_NotPropertyStatementInBlock()
+            Assert.False(IsMethodLevelMember("
+Class C
+    [|Property x As Integer|]
+        Get
+            Return 42
+        End Get
+        Set (value As Integer)
+        End Set
+    End Property
+End Class"))
+        End Sub
+
+        <Fact, WorkItem(4311, "https://github.com/dotnet/roslyn/issues/4311")>
+        Public Sub IsMethodLevelMember_FieldLikeEvent()
```

Comment:
Added (and removed `WorkItem` attributes, since this isn't associated with that issue anymore).